command/job_init.go

```diff
@@ -340,6 +340,52 @@ job "example" {
       size = 300
     }
 
+
+    # The "affinity" stanza enables operators to express placement preferences
+    # based on node attributes or metadata.
+    #
+    # For more information and examples on the "affinity" stanza, please
+    # see the online documentation at:
+    #
+    #     https://www.nomadproject.io/docs/job-specification/affinity.html
+    #
+    # affinity {
+    # attribute specifies the name of a node attribute or metadata
+    #   attribute = "${node.datacenter}"
+    #
+    # value specifies the desired attribute value. In this example Nomad
+    # will prefer placement in the "us-west1" datacenter.
+    #   value  = "us-west1"
```

Comment:
this seems to make the example GCP specific.

do you think "dc2"/"dc3" would be a simpler hint?

Reply:
since its commented out I don't have a strong opinion, its meant to be an example.